### DISCUSSION TITLE
The requested URL returned error: 403 Forbidden

### DISCUSSION THREAD

**Discussion by moonlitusun** [Homebrew/discussions#2596] [Everyday usage] [answered]
when i install fig or brew update。

brew output Error: Failed to download resource "libyaml"

I refer to https://github.com/Homebrew/discussions/discussions/962: set HOMEBREW_BOTTLE_DOMAIN=https://dl.bintray.com/homebrew

But it still not work.

**Comment by gromgit** [comment, marked as answer]
Homebrew stopped using Bintray to distribute bottles quite some time ago, so your setup sounds pretty old. What's the output of `brew config` and `brew doctor`?

**Reply by moonlitusun** [reply]
![image](https://user-images.githubusercontent.com/22940470/144780737-bdd4c68b-8c0e-4a59-a034-aec6e98757fe.png)
thanks, got it, i haven’t upgraded for a long time。
